### DISCUSSION TITLE
How to place a uwp or winui app in system tray

### DISCUSSION THREAD

**Discussion by amit7775** [unoplatform/uno#8139] [Q&A]
how to tuck the uwp app in system tray

**Comment by jeromelaban** [comment]
UWP does not have such an API at this time. Just to be sure, are you running the `.UWP` project of your solution, or another target?

**Reply by amit7775** [reply]
yes i am using .uwp project. can i tuck app in system tray using some lines of code.

**Reply by jeromelaban** [reply]
No, it's not possible, there's no API for this. WinUI 3 and WinAppSDK allow for this, though.

**Reply by contact2vikas** [reply]
How can we tuck uno, WinUI 3 and WinAppSDK aplication in system tray.
